auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java

```diff
@@ -61,6 +65,18 @@ public void cleanupVirtualGroups(String accountId, String environmentCrn) {
     }
 
     private String createOrGetVirtualGroup(String accountId, String environmentCrn, String right) {
+        String virtualGroup = getVirtualGroupFromUms(accountId, environmentCrn, right);
+        if (StringUtils.isEmpty(virtualGroup)) {
```

Comment:
spring's `StringUtils.isEmpty` is depracated, could you replace this with apache's common?